src/browser_action.js

```diff
@@ -244,6 +245,18 @@ browseraction.createQuickAddEvent_ = function(text, calendarId) {
   });
 };
 
+function addAlert(element, summary, link) {
+  $('.fab').fadeOut();
+  $(element).prepend(`<div class="alert-new-event event" data-url="${link}"><div class="start-time" style="background: #FFEB3B;"></div><div class="event-details"><div class="event-title" style="white-space: normal;">${chrome.i18n.getMessage('alert_new_event_added')} ${summary}</div></div></div>`).fadeIn();
```

Comment:
A better way to do this with jQuery is to build the `<div>` using jQuery itself:

```
var toastDiv = $('<div>')
    .addClass('alert-new-event event')
    .attr('data-url', linkUrl);
// etc.
```

Also, I think the current code is using the wrong style of quotes (` instead of '), which probably won’t compile.